app/controllers/health_check_controller.rb

```diff
@@ -0,0 +1,100 @@
+class HealthCheckController < HealthCheck::HealthCheckController
+  # This overwrites the original HealthCheckController from the health_check gem, in order to add
+  # a custom check agaist the utilty's API health check. Original controller:
+  # https://github.com/ianheggie/health_check/blob/rails5/lib/health_check/health_check_controller.rb
+  rescue_from ActiveRecord::RecordNotFound, with: :render_nothing_not_found
+
+  def index
+    return super if utility.blank?
+    return render_failed_utility_inactive unless utility.active
+    return super if utility.health_check_url.blank?
+
+    @utility_response = utility.utility_service.health_check
+    return super if @utility_response.code == 200
+
+    render_failed_utility_health_check
+  end
+
+  private
+
+  def utility
+    utility_code_header = request.headers['Utility-ID']
+    return if utility_code_header.nil?
+
+    utility_code = sanitized_utility_code(utility_code_header)
+    @utility ||= Utility.find_by!(code: utility_code)
+  end
+
+  def render_failed_utility_inactive
+    @utility_error = false
```

Comment:
It's neccessary?

Reply:
Are you asking about the false value?
I think is not necessary, I made some tests 👋🏻

Reply:
Yes, I think that assignment is unnecessary because if you access here (render_failed_utility_inactive), in the 95th line from this file, @utility_error will be null.

The same for @utility_disabled in render_failed_utility_health_check

Reply:
> Wait a second... Specs are missing here eyes

I add some specs, but it's difficult to test like Ugo here because we don't have models yet.
I think tests will be completed when starts a new project from this template 💪🏻